### ISSUE TITLE
Maintenance of volunteers tool

### ISSUE THREAD

**Issue by Logout22** [FOSDEM/volunteers#20]
It seems like a lot (most) of issues have been unresolved for a long time. Is this project in need of a new maintainer?

**Comment by emanuil-tolev** [comment]
Yes, I'd say so. We maintain it enough to run it year on year but there's barely any active development, only when we really need a feature. We were thinking of abandoning and using similar volunteer mgmt systems but haven't been able to put that effort in either.

**Comment by johanvdw** [comment]
@Logout22 : would you like to help?

**Comment by Logout22** [comment]
Yes, that is why I asked. :) Reading Emanuil's comment, I was under the impression that development was more or less halted. If that is not the case, I would be happy to help!

Before you put my name on any official list, though, I would try to work with the code for a bit. I would maybe try to create some sort of testing code first to better understand how the software is supposed to operate. We could then incorporate this into some sort of CI, e.g. GitHub actions or preferably some Jenkins server (do you know a project that would sponsor compute time BTW? ;) ).

I think the timing is good because after this weekend, we have one more year where no one really needs the software and we could potentially do a bigger refactoring.

**Comment by johanvdw** [comment]
I picked development up this year, but it would be nice someone else can join in as well.

I also think that creating fixtures (a minimal dataset) for testing would be a good start.

**Comment by Logout22** [comment]
Great! I still have some stuff on my desk that will likely keep me occupied until the end of February. Hence, I would get back to you by March if that is OK? At the point where we start development we could also think about establishing some sort of instant messaging channel (Matrix?). Unless you have fixed all the bugs by then... :D

**Comment by keunes** [comment]
One year later, post-FOSDEM 2023, I would say that as a volunteer it'd be great if some improvements (for which there are issues already) could be implemented 😇
